bundles/azure/aso/psql/bundle/.imgpkg/images.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
**suggestion** Could you add to the README.md to include the new Azure Postgres?